internal/transform/span_test.go

```diff
@@ -409,6 +409,7 @@ func TestTransform(t *testing.T) {
 		}
 		tc.after()
 	}
+	os.Unsetenv("REALLY_LONG_ENV")
```

Comment:
I think that it's suppose to be inside the for-loop, so it will be cleared after each test, isn't it?

Reply:
done